docs/guides/04-data-driven-visualizations-part-1.md

```diff
@@ -0,0 +1,184 @@
+
+As a rule of thumb, if it makes sense to apply numerical functions like addition or multiplication to the data, the data should be stored/encoded as numbers. Otherwise, the data should be stored/encoded as strings.
+
+##### One to one mapping. One category - one color.
+
+To create a one to one mapping between categories and colors (or any other list of values) the simplest function is [`buckets`](https://carto.com/developers/carto-vl/reference/#cartoexpressionsbuckets).
+
+Buckets allows to pick some or all categories from a categorical property in a particular order, allowing `ramp` to match those with the color list. Let's see it with an example:
+```CARTOVL_Viz
+// Suppose we have an election map with a `winner` property that contains the political party that won the region, like:
+// $geom                    $winner
+// GeometryOfRegionA        'conservatives'
+// GeometryOfRegionB        'progressives'
+// ...
+//
+// We can create a choropleth map by matching the winners of each region to one color by using buckets
+// This will create the following correspondence:
+//      'conservatives' <=> blue
+//      'progressives'  <=> red
+color: ramp(buckets($winner, ["Conservative Party", "Labour Party"]), [blue, red])
+```
+<div class="example-map">
+    <iframe
+        id="election-basic"
+        src="/developers/carto-vl/examples/maps/guides/ramp/election-basic.html"
+        width="100%"
+        height="500"
+        style="margin: 20px auto !important"
+        frameBorder="0">
+    </iframe>
+</div>
+<div style="margin-bottom: 20px !important">
+   <a href="/developers/carto-vl/examples#example-election---basic">View my source code!</a>
+</div>
+##### *Others*
+
+When working with categories, the concept of the *others bucket* arises. For example, the buckets function picks some categories, but, what happens with the unselected categories?
+
+In the previous example, we could have regions in which the 'socialist' party won. This category wasn't placed in the `buckets` function so it will fall back to the `others` bucket.
+
+The `others` bucket will be colored gray by default. However, it's possible to override this behavior by providing a third parameter to `ramp`: `ramp(buckets($winner, ['conservatives', 'progressives'], [red, blue], white)`.
+
+<div class="example-map">
+    <iframe
+        id="election-others-bucket"
+        src="/developers/carto-vl/examples/maps/guides/ramp/election-others-bucket.html"
+        width="100%"
+        height="500"
+        style="margin: 20px auto !important"
+        frameBorder="0">
+    </iframe>
+    <a href="/developers/carto-vl/examples#example-election---others-bucket">View my source code!</a>
```

Comment:
This view my source code are nice, but I think we should give them more space below

Reply:
Yes, I'm super lost with the `iframes` and the `dev-center` code. To be honest, making this has been a pain, and as you can see, it still isn't working very good.

Reply:
I think it's a good idea to include a link inside as you did, but we're not doing it now for the rest of the embedded examples (in an homogeneous way), so I'll just leave it outside for now, in a simple sentence, and that will also give us more space.